src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.ts

```diff
@@ -39,6 +40,8 @@ export class UserFormComponent implements OnInit {
   allRoles: Array<UserFormRoleModel>;
   messages = new SelectMessages({ empty: 'There are no roles.' }, this.i18n);
 
+  readonly ActionLabels = ActionLabels;
```

Comment:
You should use `ActionLabelsI18n`.

Reply:
Thanks a lot, @tspmelo, for your thorough review! I'm currently fixing all these issues, so I'll let you know when they are pushed.